docs/fdp-overview.markdown

```diff
@@ -0,0 +1,168 @@
+# Overlay Method Selection
```

Comment:
Absolutely. There's a tentative plan to rework the existing 'how it works' document into an overarching piece that would include such a description, and then have detailed sections for fastdp and sleeve. That will probably happen in a separate PR though.